lib/resource/resource.ex

```diff
@@ -4118,4 +4154,11 @@ defmodule AshGraphql.Resource do
       name
     end
   end
+
+  def primary_key_get_query(resource) do
```

Comment:
I think one of the things we'll need to do soon here is make this configurable, but the strategy of choosing an existing gql query for it to mimic is pretty neat.